GitUI/SplitterManager.cs

```diff
@@ -95,7 +95,8 @@ public void RestoreFromSettings(ISettingsSource settings)
                                 break;
                             case FixedPanel.Panel2:
                                 int panel2PrevSize = DpiUtil.Scale(prevSize, prevDpi) - DpiUtil.Scale(prevDistance, prevDpi);
-                                SetSplitterDistance(SplitterSize - panel2PrevSize + splitterWidth);
+                                const int paddingOffset = 2; // Refer to FormCommit.ctor+WorkaroundPaddingIncreaseBug
```

Comment:
@ivangrek this makes the splitter stable at the runtime again, however the tests on the CI are still unstable

Reply:
I don't notice any shifts for the "Commit Form", but sometimes there are changes for the main form.